chaos_kitten/toys_cli.py

```
@@ -0,0 +1,225 @@
+    help="Manage community attack profiles (toys)",
+    no_args_is_help=True,
+)
+
+# Default registry URL (can be overridden by env var)
+DEFAULT_REGISTRY_URL = "https://raw.githubusercontent.com/mdhaarishussain/chaos-kitten/main/toys/registry.json"
+REGISTRY_URL = os.environ.get("CHAOS_KITTEN_REGISTRY_URL", DEFAULT_REGISTRY_URL)
+
+def _fetch_registry() -> Dict[str, Any]:
+    """Fetch the toy registry from the remote URL."""
+    try:
+        req = urllib.request.Request(REGISTRY_URL, headers={'User-Agent': 'Chaos-Kitten-CLI'})
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)